server/src/main/java/org/elasticsearch/plugins/PluginInfo.java

```diff
@@ -315,6 +322,15 @@ public boolean hasNativeController() {
         return hasNativeController;
     }
 
+    /**
+     * Whether a license must be accepted before this plugin can be installed.
```

Comment:
Nit: I just noticed this now, but it applies to the other PRs as well. This comment seems no longer applicable since there is no longer an extra acceptance prompt.

Reply:
Ah yes, I've updated the text and I'll update the other branches once this PR is merged.